pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -1136,24 +1136,17 @@ func (b *MasqueradeBindMechanism) createNatRulesUsingNftables(proto iptables.Pro
 		if port.Protocol == "" {
 			port.Protocol = "tcp"
 		}
-		err = Handler.NftablesAppendRule(proto, "nat", "KUBEVIRT_POSTINBOUND",
-			strings.ToLower(port.Protocol),
-			"dport",
-			strconv.Itoa(int(port.Port)),
-			Handler.GetNFTIPString(proto), "saddr", getEnvoyInboundLoopbackAddress(proto),
-			"counter", "snat", "to", b.getGatewayByProtocol(proto))
-		if err != nil {
-			return err
-		}
 
-		err = Handler.NftablesAppendRule(proto, "nat", "KUBEVIRT_POSTINBOUND",
-			strings.ToLower(port.Protocol),
-			"dport",
-			strconv.Itoa(int(port.Port)),
-			Handler.GetNFTIPString(proto), "saddr", getLoopbackAdrress(proto),
-			"counter", "snat", "to", b.getGatewayByProtocol(proto))
-		if err != nil {
-			return err
+		for _, addr := range []string{getEnvoyInboundLoopbackAddress(proto), getLoopbackAdrress(proto)} {
```

Comment:
I think this belongs to the previous commit.

Reply:
Right! fixed

Reply:
I don't like the coupling to envoy. Maybe we can match he entire loopback address block "127.0.0.0/8"?